fix(up-parachain): improve command handling

evilrobot-01 opened the pull request:
Improves command handling and legacy zombienet flle support by:
- automatically adding zombinetnet-sdk `settings` to temp config when not specified
- defaulting to `polkadot` for relay, `polkadot-parachain` for parachain when not specified
- resolves `default_command` and `command` to local binaries when relative paths specified
- simplifies test `zombienet.toml` file

Closes #25

al3mart commented:
Getting the following output when running 

`pop up parachain -f ./zombienet.toml -p https://github.com/r0gue-io/pop-node`

```
Could not spawn network: unable to find canonical local path to specified command: `~/.cache/pop/polkadot-parachain` are you missing an argument?
```

AlexD10S commented:
> Getting the following output when running
> 
> `pop up parachain -f ./zombienet.toml -p https://github.com/r0gue-io/pop-node`
> 
> ```
> Could not spawn network: unable to find canonical local path to specified command: `~/.cache/pop/polkadot-parachain` are you missing an argument?
> ```

Did you install from this last version? `cargo install --path .`
I tried the steps of this PR and works: https://github.com/r0gue-io/pop-cli/issues/25

evilrobot-01 commented:
Rebased. Improved UX, added Dockerfile for debugging on Linux.

@al3mart forked zombienet-sdk to get more errors out. Can you try again on your machine and let me know what error you get please?

evilrobot-01 commented:
There was a bug with the downloading of polkadot binaries which should now be fixed.

al3mart reviewed (commented):
All previous errors seem to have disappear now. The flow continues all the way until building the sourced files.

At the moment of building it errors when building some frame pallets on which `pop-node` is dependent. Haven't been able to spot the issue as I am able to build `pop-node` locally from my local cloned repo and from the sourced cached files.
